jcplayer/src/main/java/com/example/jean/jcplayer/view/JcPlayerView.kt

```diff
@@ -478,14 +480,15 @@ class JcPlayerView : LinearLayout, View.OnClickListener, SeekBar.OnSeekBarChange
 //        }
     }
 
-    override fun onStartTrackingTouch(seekBar: SeekBar) {
-        if (jcPlayerManager.currentAudio != null) {
-            showProgressBar()
-        }
-    }
+    override fun onStartTrackingTouch(seekBar: SeekBar) {}
 
     override fun onStopTrackingTouch(seekBar: SeekBar) {
-        dismissProgressBar()
+        if (jcPlayerManager.currentAudio != null) {
+            jcPlayerManager.let {
+                it.seekTo(seekBar.progress)
```

Comment:
There is no need to use `let`, because the jcPlayerManager is not nullable

Reply:
@jeancsanchez I edited it, I have to study Kotlin😅